content/docs/other-guides/troubleshooting.md

```diff
@@ -112,7 +112,7 @@ kubectl -n ${NAMESPACE} describe pods ${PODNAME}
 
 ## Pods stuck in Pending state
 
-There are three pods that have 10Gi Persistent Volume Claims (PVCs) that will get stuck in pending state if they are unable to bind their PVC. The three pods are minio, mysql, and vizier-db.
+There are three pods that have Persistent Volume Claims (PVCs) (1 for 10Gi and 2 for 20Gi) that will get stuck in pending state if they are unable to bind their PVC. The three pods are minio, mysql, and vizier-db.
```

Comment:
should we just remove the size?

Reply:
That's probably best (especially if they're liable to change). Might be good to put a comment on how to find the sizes (since it seems, in my limited experience, kubeflow isn't necessarily aimed at folks who would know how to do that). Let me know if you have a preference one way or the other.

Reply:
I think we should just remove the size

Reply:
@hall Please update the PR then we can merge it.

Reply:
Thanks for the reminder, @sarahmaddox! I've updated the PR with a more (hopefully) straightforward aproach that doesn't require the user to know the sizes of the PVs.